shenyu-plugin/shenyu-plugin-dubbo/shenyu-plugin-apache-dubbo/src/main/java/org/apache/shenyu/plugin/apache/dubbo/cache/ApplicationConfigCache.java

```diff
@@ -56,16 +55,7 @@
             .removalListener(notification -> {
                 ReferenceConfig<GenericService> config = (ReferenceConfig<GenericService>) notification.getValue();
                 if (config != null) {
-                    try {
-                        Class<?> cz = config.getClass();
-                        Field field = cz.getDeclaredField("ref");
-                        field.setAccessible(true);
-                        // After the configuration change, Dubbo destroys the instance, but does not empty it. If it is not handled,
-                        // it will get NULL when reinitializing and cause a NULL pointer problem.
-                        field.set(config, null);
-                    } catch (NoSuchFieldException | IllegalAccessException e) {
-                        LOG.error("modify ref have exception", e);
-                    }
+                    config.destroy();
```

Comment:
config.destroy();  maybe have some error

Reply:
Can you tell me what's error